app/src/main/java/com/teampophory/pophory/feature/sign_up/SignUpDialogFragment.kt

```diff
@@ -0,0 +1,37 @@
+package com.teampophory.pophory.feature.sign_up
+
+import android.graphics.Color
+import android.graphics.drawable.ColorDrawable
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.DialogFragment
+import com.teampophory.pophory.databinding.FragmentSignUpDialogBinding
+
+class SignUpDialogFragment : DialogFragment() {
+
+    private var _binding: FragmentSignUpDialogBinding? = null
+    private val binding get() = _binding!!
```

Comment:
```suggestion
    private val binding by viewBinding(FragmentSignUpDialogBinding::bind)
```

이렇게 해주고 onCreateView는

```kotlin
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        dialog?.window?.setBackgroundDrawable(ColorDrawable(Color.TRANSPARENT))
        return FragmentSignUpDialogBinding.inflate(inflater, container, false).root
    }
```

이런식으로 작성하기만 하면 onDestroyView null로 해제 안해줘도 돼!